### PR TITLE
THJS 39 update packages frontend

### PR DESCRIPTION
**Pull request by bohdan-degtiar**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,8 +36,8 @@
     "eslint-plugin-unicorn": "45.0.2",
     "lint-staged": "13.1.0",
     "simple-git-hooks": "2.8.1",
-    "stylelint": "14.16.1",
-    "stylelint-config-standard": "29.0.0"
+    "stylelint": "15.7.0",
```
Comment (v0ldemar01) [inline package.json]: Revert this for appropriate task please

File: client/src/libs/components/modal/styles.module.scss [hunk 2]
```diff
@@ -21,31 +21,31 @@
   align-items: center;
 }
 
-@media only screen and (max-width: 767px) {
+@media only screen and (width >= 767px) {
```
Comment (v0ldemar01) [inline client/src/libs/components/modal/styles.module.scss]: It's not a good idea to make some changes which are not related to current task

**Review by v0ldemar01 — CHANGES_REQUESTED** [review]

File: client/src/libs/components/modal/styles.module.scss (hunk 2 above)
Comment (bohdan-degtiar) [reply client/src/libs/components/modal/styles.module.scss]: This depend to update stylelint-config-standard-scss that have new rules for this case.
I can`t commit changes if don`t fix stylelint errors(

File: package.json (hunk 1 above)
Comment (bohdan-degtiar) [reply package.json]: OK but this I need this changes for update stylelint-config-standard-scss
I can pull this changes after update root packages will be done.

File: client/src/libs/components/modal/styles.module.scss (hunk 2 above)
Comment (v0ldemar01) [reply client/src/libs/components/modal/styles.module.scss]: Ok, let's stay with it
